Parser/Parser.fs

```diff
@@ -618,20 +651,31 @@ module Parse =
                     <|>
                 (isProto3 >>. pStrMessageType_P3)
 
-            let pRpcOptions =
+            let optionStatment = 
+                str_ws1 "option" >>. pOption_ws .>> eostm
+
+            let pRpcOptionsNew =
+                betweenCurly
+                    (many optionStatment)
+
+            let pRpcOptionsOld =
                 betweenCurly
                     (sepBy pOption_ws (str_ws ","))
 
+            let pRpcOptions = pRpcOptionsNew <|> pRpcOptionsOld
```

Comment:
I'm not sure if `betweenCurly` consumes a token.  If so, then this will need an `attempt`, as in:

```
            let pRpcOptions = attempt pRpcOptionsNew <|> pRpcOptionsOld
```

We need a unit test for an `rpc` statement with old-style options to verify if this is needed or not.